programmers/난이도별/level01.두 개 뽑아서 더하기/develop-sell.js

```diff
@@ -0,0 +1,17 @@
+function solution(numbers) {
+  var answer = [];
+  
+  for(var i=0; i < numbers.length; i++)
+      {
+          for(var j= 1; j <numbers.length; j ++)
+              {
+                  if(j != i)
+                      answer.push(numbers[i] + numbers[j]);  
+              }
+         
+      }
```

Comment:
```suggestion
  for (let i = 0; i < numbers.length; i++) {
      for (let j = 1; j < numbers.length; j++) {
          if(j !== i)
              answer.push(numbers[i] + numbers[j]);  
      }
  }
```
가능하면 코딩 스타일을 일관적인 방식으로 유지해주세요!
다른 사람이 알아보기 쉽게 하려는 목적도 있고, 좀 문제가 길어지면 나중에 자기가 알아보기 힘들어서 풀이하지 못하는 경우도 생긴답니다 ㅠㅠ

Reply:
네 알겠습니다! 코드리뷰 감사합니다!